### DISCUSSION TITLE
How to get Student Developer Pack Benefits?

### DISCUSSION THREAD

**Discussion by gezalper** [community/community#66718] [GitHub Education]
### Hi everyone,

Hello,
I am currently dealing with issues about Student Developer Pack benefits. My application is approved but I don't know how to retrieve the benefits. Should I do something about it? Thanks.

**Comment by arafajs** [comment]
It usually takes 72 hours for the benefits to be activated.

**Comment by mantasu** [comment]
Hey, I had the same issue, it may still take a week or so before your benefits become active. When "Approved" tag becomes purple, you should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If you don't receive any benefits after 2-3 weeks, just open a [ticket](https://support.github.com/request/education?tags=docs-generic). It says there:
> Applications may take several weeks to be approved
